pkg/utils/kubeutils/kube_crd.go

```diff
@@ -0,0 +1,35 @@
+package kubeutils
+
+import (
+	"github.com/solo-io/solo-kit/pkg/api/v1/clients/kube/crd"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// The standard CRD we use for helm charts, based on the solo kit CRD
```

Comment:
not sure if this comment adds much or will be true in the future
I believe it is describing the return type, but that is already self-documenting